x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -61,9 +62,20 @@ import { getSettings } from './settings';
 import { normalizeKuery, escapeSearchQueryPhrase } from './saved_object';
 import { isAgentsSetup } from './agents/setup';
 import { appContextService } from './app_context';
+import { getPackagePermissions } from './epm/packages/get';
 
 const SAVED_OBJECT_TYPE = AGENT_POLICY_SAVED_OBJECT_TYPE;
 
+const DEFAULT_PERMISSIONS: PackagePermissions = {
```

Comment:
This is used as a fallback in case we cannot read the data_streams for whatever reason